Makefile

```diff
@@ -1,5 +1,5 @@
 # Versions from versions.txt
-VERSION ?= "$(shell grep -v '\#' versions.txt | grep operator= | awk -F= '{print $$2}')"
+VERSION ?= $(shell grep -v '\#' versions.txt | grep operator= | awk -F= '{print $$2}')
```

Comment:
Was there a reason this had to change?

Reply:
Tested by giving the static value. Will change it to as it is.

Reply:
This had to change because the latest tag would have quotes around it. So the image name ended up being `public.ecr.....:"latest"`